src/components/PlaylistPage/CourseInfo.js

```diff
@@ -24,12 +23,14 @@ const CourseInfo = React.createClass({
 
 CourseInfo.propTypes = {
   isStudentMode: PropTypes.bool,
-  t: PropTypes.func
+  t: PropTypes.func,
+  language: PropTypes.string
 };
 
 function mapStateToProps(state) {
   return {
-    t: getTranslator(state)
+    t: getTranslator(state),
+    language: state.language
   };
```

Comment:
Also, the content sent to the render function should be in an object `{__html: <content>}` instead of just the content. See https://facebook.github.io/react/docs/dom-elements.html#dangerouslysetinnerhtml

You could even make the getCourseInfo() function to this, and perhaps rename the function to getCourseInfoMarkup or something.